src/AtlasModules/atlas-config.bat

```diff
@@ -2065,7 +2066,22 @@ echo Adding extras bucket...
 cmd /c scoop bucket add extras
 goto finish
 
-:browser
+:choco
+echo Installing Chocolatey
+set /P c="Review Install script before executing? [Y/N]: "
+if /I "%c%" EQU "Y" curl "https://community.chocolatey.org/install.ps1" -o %windir%\AtlasModules\install.ps1 && notepad %windir%\AtlasModules\install.ps1
+if /I "%c%" EQU "N" curl "https://community.chocolatey.org/install.ps1" -o %windir%\AtlasModules\install.ps1
+powershell -NoProfile Set-ExecutionPolicy Bypass -Scope Process -Force; [System.Net.ServicePointManager]::SecurityProtocol = [System.Net.ServicePointManager]::SecurityProtocol -bor 3072; iex ((New-Object System.Net.WebClient).DownloadString('%windir%\AtlasModules\install.ps1'))
+echo Refreshing environment for Choco...
+call %windir%\AtlasModules\refreshenv.bat
+echo]
+echo Installing git...
+:: Scoop isn't very nice with batch scripts, and will break the whole script if a warning or error shows..
```

Comment:
Small detail, but the comment is wrong here

Reply:
This comment doesn't exist on my side, maybe you are looking at an older commit?

Reply:
yeah sorry it's an older commit